### ISSUE TITLE
request for support for continuous and multi-discrete action space

### ISSUE THREAD

**Issue by fokx** [facebookresearch/torchbeast#38]
It seems Ray's rllib now support continuous and multi-discrete action space, when will torchbeast? Thanks!

**Comment by heiner** [comment]
Hey fokx,

I appreciate the question. Realistically, we have moved over most of our agent code to [moolib](https://github.com/facebookresearch/moolib), so I wouldn't expect too many new features to this codebase. Adding multi-discrete action spaces however isn't hard, it's just a matter of using `nest.map` over the set of logits tensors. Continuous action spaces wouldn't be compatible with vtrace, so it depends on what exactly one would look to do.

At any rate my suggestion would be to try to implement whatever you need yourself -- that's a great way to check you understand what's going on and to not make any incorrect assumptions on what the code is doing.

**Comment by fokx** [comment]
Thanks for your response and suggestion!
